README.md

```diff
@@ -54,6 +54,7 @@ Build the binary.
 $ mkdir -p $GOPATH/src/github.com/kubernetes-incubator/external-dns
 $ cd $GOPATH/src/github.com/kubernetes-incubator/external-dns
 $ git clone https://github.com/kubernetes-incubator/external-dns.git .
+$ glide install --strip-vendor
```

Comment:
Yes, since we check in `vendor/` you should be fully up to date when doing a `git clone`.

@pigmej did you have any problems without the `glide install` step?